build/update-csv.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+YQ_MAJ_VERSION=${1:-4}
+installYQ() {
+    printf "Installing yq ..."
+    GO111MODULE=on go get github.com/mikefarah/yq/v4
+    PATH=$GOPATH/bin:$PATH
```

Comment:
It would be nice to set `GOPATH` if it not defined for example to `$HOME\go`

Reply:
oh right, my idea was to use `go env` but I forgot to that. fixed